### ISSUE TITLE
NuGet Packager Version Selection

### ISSUE THREAD

**Issue by ChristopherHaws** [Microsoft/vsts-tasks#3636]
Hello,

I was wondering if there is any particular reason that the NuGet Packager task does not have the ability to select a NuGet version to use. NuGet Installer and NuGet Publisher both have this ability, so it seems strange that NuGet Packager doesn't.

Thanks!

Chris

**Comment by memark** [comment]
Yes, please, this would be very useful. Now I have to provide my own nuget.exe just to use SemVer 2 version strings.
